BenchmarkCNV/README.md

```diff
@@ -0,0 +1,22 @@
+# BenchmarkCNV
+BenchmarkCNV.wdl is designed to evaluate the performance of Dragen CNV (Copy Number Variation) caller against GATK SV (Structural Variation) caller. 
+The pipeline is composed of three main steps:
+
+* Select Variants
+  * The pipeline first selects variants from the 1000 Genomes GATK SV callset, which serves as the ground truth for the evaluation.
+
+* Large Variant Evaluation
+  * The pipeline then uses the [wittyer](https://github.com/Illumina/witty.er) tool to evaluate the CNV variants discovered by the Dragen CNV caller and Dragen SV caller. 
+  Wittyer is a tool that is specifically designed for large variant evaluation, and it is capable of comparing CNV and SV calls. 
+  The pipeline benchmarks the results of the Dragen CNV caller and Dragen SV caller against GATK SV results to provide a comprehensive evaluation of the performance.
+
+* Format Output
+  * The final step of the pipeline uses the [Wittyer4mat](https://github.com/broadinstitute/TAG/pull/1567) script to parse the output generated by Wittyer into a well-formatted dataframe. 
+  This allows for easy interpretation and analysis of the evaluation results.
+
+## IMPORTANT NOTE
+* The provided JSON is a ready to use example JSON template of the BenchmarkCNV workflow. It's using 1000 genome as callset input and HG00513 as an example. 
```

Comment:
Can you also add sample outputs from Wittyer4mat on these inputs?

Reply:
will do it in the next commit.